dlpy/tests/test_layers.py

```diff
@@ -309,6 +310,38 @@ def test_bidirectional_block4(self):
         list1 = Bidirectional(n=[10, 20, 30], n_blocks=3).compile()
         self.assertTrue(self.sample_syntax['bidirectional3'] == list1)
 
+    def test_transpose_conv1(self):
```

Comment:
we will possibly try to add more unit tests to keep the code coverage in the way it is now, but lets start with the comments. thanks